Linux and Terminal commands/Commands For Windows.md

```diff
@@ -8,7 +8,7 @@
 | ls -l </br> [To see all files & directories] | Get-ChildItem |
 | cat > </br> [To edit] | vim |
 | cat file.txt two.txt > total.txt | cat file.txt, two.txt > total.txt |
-| cat file.txt | tr a-z A-Z > upper.txt | (cat 'file.txt').ToUpper() > upper.txt |
+| cat file.txt \| tr a-z A-Z > upper.txt | (cat 'file.txt').ToUpper() > upper.txt |
```

Comment:
its already corrected, 
we use \ to go in the new line hence in this command we are giving input in the same line so no need of adding  \ , it will give an error

if it there is not | pipe symbol then it will work, else it will give an error 

```diff
- cat file.txt \| tr a-z A-Z > upper.txt

```
if the command is like that then it will work
```bash
cat file.txt \
tr a-z A-Z > upper.txt
```

---
### command is already corrected
```bash
cat file.txt | tr a-z A-Z > upper.txt
```

you can close this pr